src/app/core/dto/link-type.ts

```diff
@@ -27,6 +28,8 @@ export interface LinkType {
   collectionCodes?: [string, string]; // TODO make compulsory
   fromCollection?: string; // TODO remove
   toCollection?: string; // TODO remove
+  from?: Resource;
+  to?: Resource;
```

Comment:
Is it not possible to do this some other way? Otherwise, we will have three representations of the same thing in this interface.

Do not worry if your solution is not nice. It will be done differently in the future. DTOs and objects representing front-end state will be two separate things after NgRx is introduced to this application.